packages/datagrid/src/components/DefaultHeaderRenderer/QualityBar.component.js

```diff
@@ -26,51 +26,118 @@ export function formatNumber(value) {
 
 function QualityBar(props) {
 	const { t = getDefaultT() } = useTranslation(I18N_DOMAIN_DATAGRID);
+	function getLibelleValue(numberOfItem) {
+		return numberOfItem > 1 ? 'values' : 'value';
+	}
+	const invalidTitleLabel = t('QUALITY_BAR_INVALID', {
+		count: props.invalid.total,
+		defaultValue: '{{total}} invalid {{values}} ({{percentage}}%)',
+		percentage: props.invalid.percentage,
+		total: formatNumber(props.invalid.total),
+		values: getLibelleValue(props.invalid.total),
+	});
+	const emptyTitleLabel = t('QUALITY_BAR_EMPTY', {
+		count: props.empty.total,
+		defaultValue: '{{total}} empty {{values}} ({{percentage}}%)',
+		percentage: props.empty.percentage,
+		total: formatNumber(props.empty.total),
+		values: getLibelleValue(props.empty.total),
+	});
+	const validTitleLabel = t('QUALITY_BAR_VALID', {
+		count: props.valid.total,
+		total: formatNumber(props.valid.total),
+		percentage: props.valid.percentage,
+		values: getLibelleValue(props.valid.total),
+		defaultValue: '{{total}} valid {{values}} ({{percentage}}%)',
+	});
 	return (
 		<div className={classNames(theme['td-quality-bar'], 'td-quality-bar')}>
 			{!!props.invalid.total && (
 				<span
 					className={classNames(theme['td-quality-bar-invalid'], 'td-quality-bar-invalid')}
 					style={{ width: `${props.invalid.percentage}%`, minWidth: `${QUALITY_BAR_MIN_WIDTH}%` }}
-					title={t('QUALITY_BAR_INVALID', {
-						count: props.invalid.total,
-						defaultValue: '{{total}} invalid values ({{percentage}}%)',
-						percentage: props.invalid.percentage,
-						total: formatNumber(props.invalid.total),
-					})}
-				/>
+					title={invalidTitleLabel}
+				>
+					{props.invalid.buttonFilter && (
+						<Action
```

Comment:
would be cleaner to use either a span or a button depending on onClick state, not both